src/io/WKBReader.cpp

```diff
@@ -427,35 +430,40 @@ WKBReader::readGeometryCollection()
 std::unique_ptr<CoordinateSequence>
 WKBReader::readCoordinateSequence(uint32_t size)
 {
-    unsigned int targetDim = 2 + (hasZ ? 1 : 0);
-    auto seq = factory.getCoordinateSequenceFactory()->create(size, targetDim);
+    uint32_t targetDim = 2 + (hasZ ? 1 : 0);
+    // auto seq = factory.getCoordinateSequenceFactory()->create(size, targetDim);
     if(targetDim > inputDimension) {
         targetDim = inputDimension;
     }
+    std::vector<geom::Coordinate> curCoords;
```

Comment:
Building this up in a vector instead of a CoordinateSequence seems like a win no matter what